### PR TITLE
Added validation to facade

### PR DESCRIPTION
**Pull request by AnnaSibirtseva**

### CODE REVIEW

File: src/main/java/com/andreev/coursework/dto/ChatDto.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package com.andreev.coursework.dto;
 
+import javax.validation.constraints.Size;
+
 public class ChatDto {
+    @Size(min = 1, max = 500)
```
Comment (LinKav20) [inline src/main/java/com/andreev/coursework/dto/ChatDto.java:6]: :sparkles:

File: src/main/java/com/andreev/coursework/dto/MessageDto.java [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
-    private String dateSend;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy HH:mm:ss")
+    private Date dateSend;
 
     public String getContent() {
         return content;
     }
 
-    public String getDateSend() {
+    public Date getDateSend() {
```
Comment (LinKav20) [inline src/main/java/com/andreev/coursework/dto/MessageDto.java:16]: perfect naming <3

Comment (AnnaSibirtseva) [reply src/main/java/com/andreev/coursework/dto/MessageDto.java:16]: <3

File: src/main/java/com/andreev/coursework/controller/ChatController.java [hunk 3]
```diff
@@ -59,17 +59,17 @@ public ResponseEntity<String> addMember(
     @PostMapping("/{chatId}/addMessage")
     @Operation(summary = "отправить сообщение в чат")
     public ResponseEntity<String> addMessage(
-        @PathVariable int chatId,
-        @RequestBody MessageDto messageDto,
-        Authentication authentication
+            @PathVariable int chatId,
+            @RequestBody MessageDto messageDto,
```
Comment (G0P0Ke) [inline src/main/java/com/andreev/coursework/controller/ChatController.java]: Add annotation for validation. I mean @Valid annotation

**Review by LinKav20 — APPROVED** [review]

File: src/main/java/com/andreev/coursework/security/SecurityController.java [hunk 4]
```diff
@@ -24,37 +28,33 @@ public class SecurityController {
...
 
     @PostMapping(
-        value = "/entry",
-        consumes = "application/json"
+            value = "/entry",
+            consumes = "application/json"
     )
     @Operation(summary = "Аутентификация пользователя")
     public ResponseEntity<JwtResponse> authenticateEmployee(@RequestBody EntryDto entryDto) {
         boolean entryFlag = participantService.isActive(entryDto.getEmail(), entryDto.getCode());
```
Comment (G0P0Ke) [inline src/main/java/com/andreev/coursework/security/SecurityController.java:54]: Lost one

**Pull request merged**